apps/mivisionx_openvx_classifier/README.md

```diff
@@ -55,35 +58,39 @@ Usage: ./classifier
 
 #### Generating weights.bin for different Models
 
-1. Download or train your own caffemodel for the supported models listed above.
+1. Download or train your caffemodel for the supported models listed above.
 
 Here is the sample download [link](https://github.com/SnailTyan/caffe-model-zoo) that contains all the prototxt: 
 
 2. Use [MIVisionX Model Compiler](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/tree/master/model_compiler#neural-net-model-compiler--optimizer) to extract weights.bin from the pre-trained caffe models
+
 **Note:** MIVisionX installs all the model compiler scripts in `/opt/rocm/mivisionx/model_compiler/python/` folder
 
+ + Convert the pre-trained caffemodel into AMD NNIR model:
+
+  ``` 
+  % python /opt/rocm/mivisionx/model_compiler/python/caffe_to_nnir.py <net.caffeModel> <nnirOutputFolder> --input-dims <n,c,h,w> [--verbose <0|1>]
+  ```
 
-	* Convert the pre-trained caffemodel into AMD NNIR model:
+  Sample:
 
-	````
-	% python /opt/rocm/mivisionx/model_compiler/python/caffe_to_nnir.py <net.caffeModel> <nnirOutputFolder> --input-dims <n,c,h,w> [--verbose <0|1>]
-	````
-		Sample:
-		````
-		% python /opt/rocm/mivisionx/model_compiler/python/caffe_to_nnir.py VGG_ILSVRC_16_layers.caffemodel VGG16_NNIR --input-dims 1,3,224,224
-		````
+    ``` 
+    % python /opt/rocm/mivisionx/model_compiler/python/caffe_to_nnir.py VGG_ILSVRC_16_layers.caffemodel VGG16_NNIR --input-dims 1,3,224,224
+    ```
 
-	* Convert an AMD NNIR model into OpenVX C code:
+ + Convert an AMD NNIR model into OpenVX C code:
 
-	````
-	% python /opt/rocm/mivisionx/model_compiler/python/nnir_to_openvx.py <nnirModelFolder> <nnirModelOutputFolder>
-	````
-		Sample:
-		````
-		% python /opt/rocm/mivisionx/model_compiler/python/nnir_to_openvx.py VGG16_NNIR VGG16_OpenVX
-		````
+  ``` 
+  % python /opt/rocm/mivisionx/model_compiler/python/nnir_to_openvx.py <nnirModelFolder> <nnirModelOutputFolder>
+  ```
 
-	**Note:** The weights.bin file will be generated inside the OpenVX folder and you can use that as an input for this project.
+  Sample:
+
+    ``` 
+    % python /opt/rocm/mivisionx/model_compiler/python/nnir_to_openvx.py VGG16_NNIR VGG16_OpenVX
+    ```
+
+ **Note:** The weights.bin file will be generated inside the OpenVX folder and you can use that as an input for this project.
```

Comment:
Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, add 2 spaces](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6056700)